ocw/lib/provider.py

```diff
@@ -7,12 +7,22 @@
 import logging
 
 
+class ProviderAdapter(logging.LoggerAdapter):
```

Comment:
this class would make sense if you will use it not only for provider's ancestors . please replace all other loggers with it . also in such a case it would be not right to have it together with provider . please move it to separate file

Reply:
please ignore this